modules/standard/UtilReplicatedVar.chpl

```diff
@@ -115,10 +144,18 @@ proc rcCollect(replicatedVar: [?D] ?MYTYPE, collected: [?CD] MYTYPE): void
       col = replicatedVar[rcDomainIx];
 }
 
+/*
+Access the copy of `replicatedVar` on the current locale.
+
+This is equivalent to ``replicatedVar[1]``.
```

Comment:
If so, that strikes me as an implementation detail we don't want to reveal.  Since we're hiding rcDomainIx from the user, and hiding the inner working of rcLocal, it seems consistent to me to just remove that line.